README.md

```diff
@@ -3,14 +3,20 @@
 > A full-featured markdown parser and compiler, written in JavaScript. Built
 > for speed.
 
-[![NPM version](https://badge.fury.io/js/marked.png)][badge]
+[![NPM version](https://badge.fury.io/js/marked.svg)][badge]
 
```

Comment:
Hmm... I do not understand why this change is shown, since at the moment it [already exists](https://raw.githubusercontent.com/chjj/marked/18deb8b07128d37ac2e2be01d4efbe3423fd7ecb/README.md) in the master.

Reply:
You probably rebased (wrong?)

Reply:
It looks like you rebased onto 43fc870c4e441d14e23cf9e8df30b990dbc36987
instead of the latest commit.

Make sure you fetch the latest chjj/master branch before rebasing on it.